File: programacao/python/respostas/selecao/20.py
import random

usuario = int(input("Digite 1 (papel), 2 (pedra) ou 3 (tesoura): "))
computador = random.randint(1, 3)

print(f"Computador escolheu: {computador}")

if usuario == computador:
    print("Empate")
elif (
    (usuario == 1 and computador == 2)
    or (usuario == 2 and computador == 3)
    or (usuario == 3 and computador == 1)
):
    print("Voce venceu")
else:
    print("Computador venceu")

File: programacao/c/respostas/selecao/20.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void) {
    int usuario, computador;

    printf("Digite 1 (papel), 2 (pedra) ou 3 (tesoura): ");
    scanf("%d", &usuario);

    srand(time(NULL));
    computador = (rand() % 3) + 1;

    printf("Computador escolheu: %d\n", computador);

    if (usuario == computador) {
        printf("Empate\n");
    } else if ((usuario == 1 && computador == 2) ||
               (usuario == 2 && computador == 3) ||
               (usuario == 3 && computador == 1)) {
        printf("Voce venceu\n");
    } else {
        printf("Computador venceu\n");
    }

    return 0;
}